### PR TITLE
updating contributor instructions

### PR DESCRIPTION
**Pull request by zbtirrell**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,9 +46,9 @@ that with the npm `-g` flag.
...
 ```
 
-#### Install all the things
+#### Install all the NPM packages
 
-With all of those in place, simply run:
+With all of those in place, run:
 
 ```
 npm install
```
Comment (lucatume) [inline CONTRIBUTING.md]: It's worth mentioning we use version `8.9.4`.
If one does not have that (that's old) then `nodenv` should be used.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -53,6 +61,27 @@ rebuild and try again.
...
+composer install
+```
+
+#### Install npm and composer packages in common
+
+One of our submodules is Tribe Common, which is a shared set of libraries in use by our plugins (see also [Event Tickets](https://github.com/moderntribe/event-tickets)). It uses npm and composer as well, so, let's get those as well.
+
+```
+cd common
+npm install
```
Comment (lucatume) [inline CONTRIBUTING.md]: Same note about `node` versions here.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -53,6 +61,27 @@ rebuild and try again.
...
+composer install
```
Comment (lucatume) [inline CONTRIBUTING.md]: `common` has sub-modules too:
```suggestion
composer install
git submodule update --init --recursive
```

Comment (zbtirrell) [reply CONTRIBUTING.md]: the --recursive should cover that though, right?

Comment (borkweb) [reply CONTRIBUTING.md]: Yup, it will. We don't need the `git submodule update --init --recursive` here.

**Pull request merged**